### ISSUE TITLE
Using multiple aspects causes host player's character model becoming invisible to clients

### ISSUE THREAD

**Issue by UnknownMaster21** [William758/ZetAspects#24]
Here's the pic from client's pespective. Pasted into discord, so here's the link: https://media.discordapp.net/attachments/316204332557008906/892872069103104020/unknown.png?width=1168&height=657

Another theory may be with custom skins used, but let's give a shot anyway. ( Yes, this is also test run meanwhile )

**Comment by William758** [comment]
Yeah I cant think of anything I have done that would cause the model to disappear. Good thing it seems to fix itself though. My mod will try to show the EliteVariety skins first before the vanilla t1 ones so maybe it does have something to do with how they made their material.
